install/Makefile

```diff
@@ -142,6 +143,13 @@ export HUGO_VERSION ?= 0.45.1
 hugo: hugo-$(OS)
 	@exit 0
 
+export LECTL_VERSION ?= 0.16
+# Releases: https://github.com/sahsanu/lectl/releases
+## Install lectl script to check issued certificates by Let's Encrypt on CTL
+lectl:
```

Comment:
Add -sSL arguments so we properly handle redirects and failures

Reply:
`$(CURL)` it's `CURL ?= curl --fail -sSL`

Reply:
Oops. You're right =)